### ISSUE TITLE
`add https:\\...` failure on windows

### ISSUE THREAD

**Issue by tshort** [JuliaLang/Pkg.jl#366]
I'm getting a failure on Windows (alpha binary). This works fine in Linux. I tried a couple of different URLs and got the same error.

```julia
(v0.7) pkg> add https://github.com/jrevels/Cassette.jl.git
ERROR: stat: invalid argument (EINVAL)
Stacktrace:
 [1] stat(::String) at .\stat.jl:68
 [2] isdir at .\stat.jl:303 [inlined]
 [3] #parse_package#5(::Pkg.REPLMode.CommandKind, ::Function, ::String) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v0.7\Pkg\src\REPLMode.jl:145
 [4] #parse_package at .\<missing>:0 [inlined]
 [5] do_add_or_develop!(::Pkg.Types.Context, ::Array{Union{Pkg.Types.VersionRange, String, Pkg.REPLMode.Command, Pkg.REPLMode.Option, Pkg.REPLMode.Rev},1}, ::Pkg.REPLMode.CommandKind) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v0.7\Pkg\src\REPLMode.jl:573
 [6] #invokelatest#1 at .\essentials.jl:670 [inlined]
 [7] invokelatest at .\essentials.jl:669 [inlined]
 [8] do_cmd!(::Array{Union{Pkg.Types.VersionRange, String, Pkg.REPLMode.Command, Pkg.REPLMode.Option, Pkg.REPLMode.Rev},1}, ::REPL.LineEditREPL) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v0.7\Pkg\src\REPLMode.jl:275
 [9] #do_cmd#8(::Bool, ::Function, ::REPL.LineEditREPL, ::String) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v0.7\Pkg\src\REPLMode.jl:235
 [10] do_cmd at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v0.7\Pkg\src\REPLMode.jl:232 [inlined]
 [11] (::getfield(Pkg.REPLMode, Symbol("##27#30")){REPL.LineEditREPL,REPL.LineEdit.Prompt})(::REPL.LineEdit.MIState, ::Base.GenericIOBuffer{Array{UInt8,1}}, ::Bool) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v0.7\Pkg\src\REPLMode.jl:950
 [12] top-level scope
```

**Comment by pfitzseb** [comment]
Probably fixed by https://github.com/JuliaLang/Pkg.jl/pull/323.

**Comment by tshort** [comment]
How does one update `Pkg` to test? I tried `dev Pkg`, but that didn't work.

**Comment by KristofferC** [comment]
Fixed by #323
